.github/workflows/htmltest.yml

```diff
@@ -0,0 +1,57 @@
+# .github/workflows/htmltest.yml
+# Run htmltest link checker on generated HTML output
+name: Link check with htmltest
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - trunk
+  pull_request:
+
+jobs:
+  htmltest:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: "latest"
```

Comment:
I missed this. I prefer to pin the Hugo version and upgrade when we have time to check on things. What is your experience with this?

Reply:
> I missed this. I prefer to pin the Hugo version and upgrade when we have time to check on things. What is your experience with this?

Yes, that is of course the better setting in your scenario. Changed now.